apps/api/infra/api.ts

```diff
@@ -181,6 +181,18 @@ export const serviceSetup = (services: {
       },
       HSN_WEB_FORM_ID: '1dimJFHLFYtnhoYEA3JxRK',
       SESSIONS_API_URL: ref((h) => `http://${h.svc(services.sessionsApi)}`),
+      AUTH_ADMIN_API_URL_DEV:
+        'http://identity-server-admin.services-auth-admin-api.svc.cluster.local',
+      AUTH_ADMIN_API_URL_STAGING: {
+        dev: '',
+        staging: 'https://identity-server-admin.staging01.devland.is',
+        prod: 'https://identity-server-admin.staging01.devland.is',
+      },
+      AUTH_ADMIN_API_URL_PROD: {
+        dev: '',
+        staging: '',
+        prod: 'https://identity-server-admin.innskra.island.is',
+      },
```

Comment:
@island-is/devops  I need help here to configure the URLs and then we need to do some AWS magic to query cross environments 😅

cc @busla

Reply:
```
DEV: https://services-auth-admin-api.internal.dev01.devland.is
STAGING: https://services-auth-admin-api.internal.staging01.devland.is
PROD: https://services-auth-admin-api.internal.innskra.island.is
```

Let's go with this until we know more.